site/content/en/docs/Reference/api.md

```diff
@@ -689,9 +689,9 @@ messages are not finalized and still subject to possible change or removal.
 | ----- | ---- | ----- | ----------- |
 | id | [string](#string) |  | Id represents an auto-generated Id issued by Open Match. |
 | search_fields | [SearchFields](#openmatch.SearchFields) |  | Search fields are the fields which Open Match is aware of, and can be used when specifying filters. |
-| extensions | [Backfill.ExtensionsEntry](#openmatch.Backfill.ExtensionsEntry) | repeated | Customized information not inspected by Open Match, to be used by the match making function, evaluator, and components making calls to Open Match. Optional, depending on the requirements of the connected systems. |
+| extensions | [Backfill.ExtensionsEntry](#openmatch.Backfill.ExtensionsEntry) | repeated | Customized information not inspected by Open Match, to be used by the MatchMakingFunction, evaluator, and components making calls to Open Match. Optional, depending on the requirements of the connected systems. |
```

Comment:
To be a bit more in tune with the rest of the docs, if I may suggest, probably using "Match Function" here. Open to keeping it the same.

Reply:
you mean `Match Making Function`, right??

Reply:
```MatchMakingFunction``` -> ```MatchFunction```
Yeah that is what I was referring to. I assume you're speaking about Match Function service in the same context you referred to the evaluator so I figured this change would be a bit more consistent.

Reply:
oh, i see. Ok, no problem, will change it then